settings.py

```diff
@@ -1089,6 +1089,8 @@ def _pkce_required(client_id):
 ALLOW_PHONE_AS_DEFAULT_TWO_FACTOR_DEVICE = False
 RATE_LIMIT_SUBMISSIONS = False
 
+DATA_UPLOAD_MAX_NUMBER_FILES = None
```

Comment:
My read is that this reverts to the previous behavior (before the newer version of Django), which was to set no limit. Is that right?

Reply:
Correct, it was introduced with 3.2.18 and the previous behavior is equivalent to the setting set to None.
https://docs.djangoproject.com/en/4.1/ref/settings/#data-upload-max-number-files